### ISSUE TITLE
TypeError when running the environment setup part of a simulation like in the documentation

### ISSUE THREAD

**Issue by Nosudrum** [tudat-team/tudat-space#4]
I am trying to run a simulation like the one [shown in the documentation](https://tudat-space.readthedocs.io/en/latest/_src_first_steps/simulations/example_application_2.html) and I get the following error : 
```
TypeError: get_default_body_settings(): incompatible function arguments. The following argument types are supported:
    1. (bodies: List[str], initial_time: float, final_time: float, time_step: float) -> Dict[str, tudatpy.kernel.simulation.environment_setup.BodySettings]
    2. (bodies: List[str]) -> Dict[str, tudatpy.kernel.simulation.environment_setup.BodySettings]

Invoked with: ['Earth'], 631108800.0, 633787199.0, 'Earth', 'J2000'
```

Could you please let me know what I should change in the environment setup code to make it work ?

Thanks,

Arnaud Muller

**Comment by transferorbit** [comment]
Which version of `TudatPy` are you using? This might be fixed in `0.5.21`. I won’t close this issue though until you confirm the latest version fixes the issue.

Installing or updating to a new version should be as straightforward as `conda install tudatpy=0.5.21` or `conda update tudatpy=0.5.21`

**Comment by Nosudrum** [comment]
I am not sure which version I was using, but it's definitely fixed after using `conda install tudatpy=0.5.21`.

Thanks for the help !

**Comment by transferorbit** [comment]
Good to hear — thanks for the feedback!
